src/main/java/me/karakelley/http/RequestValidator.java

```diff
@@ -0,0 +1,27 @@
+package me.karakelley.http;
+
+import java.util.Arrays;
+
+public class RequestValidator {
+  private String[] methods = new String[]{"GET", "POST"};
+
+  public boolean isValid(Request request) {
```

Comment:
I really like that you separated this out into its own class. I wonder if it'd be worth adjusting it a bit so that `RequestValidator` operates on some temporary, pending request object—the idea being that your system cannot even create an invalid `Request` object at all. Maybe something with the Builder? I'm genuinely not sure, and maybe now is not the time to go overboard with it. But something to consider.